prow/userdashboard/userdashboard.go

```diff
@@ -168,24 +195,50 @@ func (da *DashboardAgent) HandleUserDashboard(queryHandler PullRequestQueryHandl
 
 // Query function that returns a list of open pull requests owned by the user whose access token
 // is consumed by the github client.
-func (da *DashboardAgent) Query(ctx context.Context, ghc githubClient) ([]PullRequest, error) {
-	var prs = []PullRequest{}
+func (da *DashboardAgent) Query(ctx context.Context, ghc githubClient, query string) ([]PullRequest, error) {
```

Comment:
Rename this to distinguish it from `ghc.Query` which is not PR specific. Maybe `QueryPullRequests`?

Reply:
done